### PR TITLE
[Feat/Fix] Updated frontend packages

### LINKED ISSUE #315: Update Antd 4.2.5
**Is your feature request related to a problem? Please describe.**
This should be updated to resolve known bugs in antd

**Describe the solution you'd like**
just update the package

### LINKED ISSUE #503: Update frontend node js packages
The following packages need to be updated and tested: 

- @ant-design/compatible         ^1.0.2  →    ^1.0.4 
- @ant-design/icons              ^4.0.5  →    ^4.2.1 
- @reduxjs/toolkit               ^1.3.6  →    ^1.4.0 
- @sentry/browser               ^5.15.5  →   ^5.19.1 
- @testing-library/jest-dom       5.8.0  →    5.11.0 
- @testing-library/react        ^10.0.4  →   ^10.4.5 
- @testing-library/user-event    10.3.2  →   12.0.11 
- antd                           ^4.1.4  →    ^4.4.1 
- bizcharts                      ^3.5.8  →    ^4.0.9 
- customize-cra                  ^0.9.1  →    ^1.0.0 
- history                       ^4.10.1  →    ^5.0.0 
- keycloak-js                   ^10.0.1  →   ^10.0.2 
- less                          ^3.11.1  →   ^3.11.3 
- less-loader                    ^5.0.0  →    ^6.2.0 
- lodash                       ^4.17.15  →  ^4.17.19 
- moment                        ^2.24.0  →   ^2.27.0 
- query-string                  ^6.12.1  →   ^6.13.1 
- react-app-rewired              ^2.1.5  →    ^2.1.6 
- react-intl                     ^4.5.5  →    ^5.0.2 
- react-scripts                   3.4.0  →     3.4.1 
- eslint                          6.8.0  →     7.4.0
- eslint-config-airbnb           18.1.0  →    18.2.0
- eslint-config-node             ^4.0.0  →    ^4.1.0
- eslint-plugin-import          ^2.20.2  →   ^2.22.0
- eslint-plugin-jsx-a11y          6.2.3  →     6.3.1
- eslint-plugin-prettier         ^3.1.3  →    ^3.1.4
- eslint-plugin-react           ^7.20.0  →   ^7.20.3
- eslint-plugin-react-hooks      ^4.0.2  →    ^4.0.6

### PR DESCRIPTION
**Pull request by BenJeau**
Changes
- Removed unnecessary ant design compatible packages
- Moved testing packages to dev dependencies
- Updated packages that don't break the app

fixes #503 
fixes #315 
closes #552

### CODE REVIEW

File: services/frontend-v3/package.json [hunk 1]
```diff
@@ -3,31 +3,28 @@
...
-    "@reduxjs/toolkit": "^1.3.6",
-    "@testing-library/jest-dom": "5.8.0",
-    "@testing-library/react": "^10.0.4",
-    "@testing-library/user-event": "10.3.2",
-    "antd": "^4.1.4",
+    "@ant-design/icons": "^4.2.1",
+    "@reduxjs/toolkit": "^1.4.0",
+    "@testing-library/jest-dom": "5.11.1",
+    "antd": "^4.4.3",
     "axios": "^0.19.2",
```
Comment (mo-5) [inline services/frontend-v3/package.json:10]: does updating axios break the app? I think we are very behind with that version

**Review by mo-5 — APPROVED** [review]

**Review by TrevorBivi — APPROVED** [review]

File: services/frontend-v3/package.json (hunk 1 above)
Comment (BenJeau) [reply services/frontend-v3/package.json:10]: I don't think I've updated it

Comment (mo-5) [reply services/frontend-v3/package.json:10]: I am not sure if you have seen this before https://www.npmjs.com/package/npm-check-updates but it's helpful to use for finding updates. Simply by running  `npx ncu`

Comment (BenJeau) [reply services/frontend-v3/package.json:10]: Ohh, I was just hovering on the packages in package.json, and it told me the latest version... that sounds like a better way to do it haha

**Pull request merged**